<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vector Split</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLineEdit" name="outShape">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="toolOut">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Unique ID field</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QComboBox" name="inField"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QComboBox" name="inShape"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Output folder</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Input vector layer</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter</set>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QDialogButtonBox" name="buttonBox_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox_2</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox_2</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
